webview-ui/src/components/chat/ChatRow.tsx

```diff
@@ -323,36 +323,14 @@ export const ChatRowContent = ({
 			case "searchAndReplace":
 				return (
 					<>
-						<div className="flex items-center gap-2.5 mb-2.5">
+						<div style={headerStyle}>
 							{toolIcon("replace")}
-							<span className="font-bold">
+							<span style={{ fontWeight: "bold" }}>
```

Comment:
should we use tailwind here ?

Reply:
Looks like it just made it consistent with the other chatrows. Maybe we should update them all as a follow-up?